communications/models.py

```diff
@@ -1,3 +1,69 @@
+from django.contrib.auth import get_user_model
+from django.core.exceptions import ValidationError
 from django.db import models
 
-# Create your models here.
+User = get_user_model()
+
+
+class ChatRoom(models.Model):
+    """
+        ChatRoom represents a room where a startup and an investor communicate.
+
+        Attributes:
+            investor (ForeignKey): The investor involved in the conversation.
+            startup (ForeignKey): The startup involved in the conversation.
+            created_at (DateTimeField): Timestamp when the chat room was created.
+        """
+    investor = models.ForeignKey(User, on_delete=models.SET_NULL, related_name='investor_chatrooms')
+    startup = models.ForeignKey(User, on_delete=models.SET_NULL, related_name='startup_chatrooms')
```

Comment:
When using `on_delete=models.SET_NULL`, it’s important to mark these fields as `null=True`. Otherwise, it can cause migration issues.

Reply:
done